rest/handler.go

```diff
@@ -112,7 +113,7 @@ const (
 
 type handlerMethod func(*handler) error
 
-// Creates an http.Handler that will run a handler with the given method
+// Creates a http.Handler that will run a handler with the given method
```

Comment:
I'm going to say that whoever writes the comment in the first place gets to set the terms of the 'aitch vs. haitch' debate, so these should stay as 'an'.

Reply:
Okay 😛 Goland said they were all wrong so just went with tweaking it